samples/matter/light_bulb/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ get_filename_component(MATTER_MODULE_ROOT $ENV{ZEPHYR_BASE}/../modules/lib/matte
 set(mcuboot_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.mcuboot.root)
 set(multiprotocol_rpmsg_KCONFIG_ROOT ${MATTER_MODULE_ROOT}/config/nrfconnect/chip-module/Kconfig.multiprotocol_rpmsg.root)
 
+# Propagate nRF7002 EK shield configuration to the child image to include Wi-Fi coexistence definitions.
+if (SHIELD STREQUAL "nrf7002_ek")
+    set(hci_rpmsg_SHIELD "nrf7002_ek")
```

Comment:
There's actually a separate shield that only defines the coexistence interface needed by the nRF5340 network core: https://github.com/nrfconnect/sdk-nrf/blob/main/boards/shields/nrf7002_ek/nrf7002_ek_coex.overlay. Please use it instead for the child image

Reply:
Done